src/utils/decancer.rs

```diff
@@ -31,7 +33,10 @@ pub async fn cure(
         "",
     );
 
-    if cured_name.is_empty() || !cured_name.starts_with(|c: char| c.is_ascii_alphabetic()) {
+    if cured_name.is_empty()
+        || !cured_name.starts_with(|c: char| c.is_ascii_alphabetic())
+        || censor.check(&cured_name)
```

Comment:
Is this supposed to be reformatted? (check with clippy)

Reply:
Does this repository not feature a workflow with clippy?

Reply:
It does and treats as an error.